examples/secrets/secret-orchestrator-tls-org-3-client-pair.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: org-3
+  namespace: org-3
+spec:
+  secretName: orchestrator-tls-client-pair
+  duration: 2160h
+  renewBefore: 360h # 15 days
+  subject:
+    organizations:
+      - MyOrg3MSP
+  isCA: false
+  privateKey:
+    algorithm: RSA
+    encoding: PKCS1
+    size: 2048
+  usages:
+    - server auth
+    - client auth
+  dnsNames:
+    - substra-backend.org-1.com
```

Comment:
`org-1`? 🤔

Reply:
c'est pareil dans le fichier `tls-org-2)client-pair.yaml` donc j'ai juste dupliquer.... Mais p-e que c'est une erreur dans le fichier org-2 enfaite ?

Reply:
I'm not sure where these values are injected however I don't see why it should be pointing to a different organization (naively).

Reply:
My neither (but even more naively...) @oleobal do you have an opinion on this ?